sbnanaobj/StandardRecord/SRSlice.h

```diff
@@ -46,6 +46,8 @@ namespace caf
 
       FlashMatch fmatch; //!< Optical flash-match for this slice of TPC charge
 
+      SRFakeReco freco;
```

Comment:
I think this name is probably too terse, and next to `fmatch` it makes it seem like it must mean `flash reconstruction`. Can we change to `fake_reco`? That also matches what it is called in the top-level StandardRecord object.